icons/couchbase/couchbase-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M 107.257812 75.222656 C 107.429688 79.167969 104.597656 82.617188 100.671875 83.238281 C 93.089844 84.539062 77.285156 85.359375 64.003906 85.359375 C 50.726562 85.359375 34.917969 84.558594 27.328125 83.238281 C 22.988281 82.492188 20.742188 79.039062 20.742188 75.222656 L 20.742188 50.222656 C 20.789062 46.351562 23.519531 43.023438 27.328125 42.199219 C 31.140625 41.6875 34.980469 41.4375 38.828125 41.449219 C 40.398438 41.492188 41.644531 42.785156 41.617188 44.347656 L 41.617188 61.960938 C 49.417969 61.960938 56.160156 61.503906 64.019531 61.503906 C 71.882812 61.503906 78.566406 61.960938 86.398438 61.960938 L 86.398438 44.492188 C 86.285156 42.929688 87.464844 41.570312 89.035156 41.457031 L 89.179688 41.457031 C 93.023438 41.445312 96.867188 41.695312 100.675781 42.203125 C 104.480469 43.03125 107.210938 46.355469 107.257812 50.222656 Z M 64.003906 0 C 28.65625 0 0 28.660156 0 64 C 0 99.347656 28.65625 128 64 128 C 99.347656 128 128 99.347656 128 64 C 128 28.660156 99.34375 0 64 0 Z M 64.003906 0 "/></svg>
```

Comment:
This can be removed since it's the same icon as `couchbase.original.svg`, but without the color. So instead of duplicating the icon, we just make an alias like you already did in `devicon.json`
![image](https://github.com/devicons/devicon/assets/43886029/8cc4ddfe-5393-4f9b-8ada-e272753517c7)

Reply:
I have removed couchbase-plain.svg and made changes in devicon.json accordingly.